src/python/visclaw/data.py

```diff
@@ -300,7 +302,7 @@ def getgauge(self, gaugeno, outdir=None, verbose=True):
                 # solutions below
                 locations = {}
                 for gauge in gauge_data.gauges:
-                    locations[gauge[0]] = gauge[1:3]
+                    locations[gauge[0]] = gauge[1:1+self.num_dim]
```

Comment:
I think we could avoid `num_dim` by changing this to 
       `locations[gauge[0]] = gauge[1:-2]`
since the location `x,y` or `x,y,z` is always followed by exactly two values `t1` and `t2`.